draft-ietf-masque-h3-datagram.md

```diff
@@ -194,26 +207,114 @@ recent draft version from the intersection set. This ensures that both
 endpoints negotiate the same draft version.
 
 
-# Capsules {#capsule}
 
-This specification introduces the Capsule Protocol. The Capsule Protocol is a
-sequence of type-length-value tuples that new HTTP Upgrade Tokens (see {{Section
-16.7 of !HTTP=I-D.ietf-httpbis-semantics}}) can choose to use. It allows
-endpoints to reliably communicate request-related information end-to-end on HTTP
-request streams, even in the presence of HTTP intermediaries. The Capsule
-Protocol can be used to exchange HTTP Datagrams when HTTP is running over a
-transport that does not support the QUIC DATAGRAM frame.
+## The DATAGRAM Capsule {#datagram-capsule}
```

Comment:
Closely match "HTTP/3 Datagrams" header
```suggestion
## HTTP/2 and HTTP/1.1 Datagrams using DATAGRAM Capsules {#datagram-capsule}
```

Reply:
again, this implies a version restriction the doesn't exist